extra/mender-client-docker-addons/Dockerfile

```diff
@@ -25,7 +27,19 @@ WORKDIR /src/mender
 RUN make prefix=/mender-install install
 RUN jq ".ServerCertificate=\"/usr/share/doc/mender-client/examples/demo.crt\" | .ServerURL=\"https://docker.mender.io/\"" \
     < examples/mender.conf.demo > /mender-install/etc/mender/mender.conf
-RUN echo artifact_name=original > /mender-install/etc/mender/artifact_info
+
+# TODO - Should this only handle the artifact_name in the Bootstrap Artifact, or checksum as well (?)
```

Comment:
This client does not support rootfs-image updates, so `artifact_name` and  a dummy `rootfs-image.version` is enough :+1: 

Please clean-up the comment

Reply:
good, thx!